packages/components/inputs/date-range-input/src/date-range-input.js

```diff
@@ -335,6 +335,9 @@ class DateRangeCalendar extends React.Component {
                 <CalendarBody
                   inputRef={this.inputRef}
                   inputProps={getInputProps({
+                    // Unset the aria-labelledby as it interfers with the link
+                    // between the <label for> and the <input id>.
+                    'aria-labelledby': undefined,
```

Comment:
@jonnybel this fixes the issue with RTL and the a11y selector.

Reply:
Oh, so downshift no longer adds the attribute if we set it to undefined? Great find!

Reply:
The other way around: downshift sets the `aria-labelledby`, but the `input` ends up with having both the `id` and the `aria-labelledby`, which I suppose confuses RTL.

Reply:
Yeah I see... But it's still weird that when I tried to reproduce that rendered markup in a codesandbox, RTL didn't complain, which made me believe it wasn't the problem 🤷‍♂️

Reply:
Did you also use downshift?

Reply:
No, I used the markup rendered by it. That was the mistake, I guess.

Reply:
I opened a separate PR with the fix: https://github.com/commercetools/ui-kit/pull/1497